hedera-node/src/test/java/com/hedera/services/ServicesStateTest.java

```diff
@@ -160,7 +170,10 @@
 	Map<TransactionID, TxnIdRecentHistory> txnHistories;
 	NetworkCtxManager networkCtxManager;
 
-	ServicesState subject;
+	private LogCaptor logCaptor;
```

Comment:
I actually think it would be worth a small comment here (and each time we define private LogCaptor logCaptor) that it is being injected by the extension. New people coming on will be quite confused.

Reply:
Alternatively, is there another way to handle this that makes it more clear? For example, can the LogCaptureExtension be written so that it only does the injection if the field is annotated with @Inject? In this way, the code will become self documenting and we can avoid having to write documentation each time and we can avoid confusing new people who are not sure how this field acquired its value.

Reply:
Makes sense! Updated `LogCaptorExtension` to now require `javax.inject.Inject` on the `LogCaptor` field; definitely more readable, and the IDE doesn't highlight the field as uninitialized now. 😇